AddrMgr/AddrIA.cpp

```diff
@@ -302,6 +316,13 @@ unsigned long TAddrIA::getT2Timeout() {
     }
 
     x  = (unsigned long)time(NULL);
+    struct timespec clockts;
+    if (clock_gettime(CLOCK_MONOTONIC_RAW, &clockts) == 0) {
+        //The time we are taking is with reference to boot time,
+        //which is not changed wvwn when the NTP updates the latest time.
+        ts = (unsigned long)clockts.tv_sec;
```

Comment:
Wrong variable.

Reply:
@armcc you're commenting on a PR that's been already merged. If you feel it's not right, please open a new PR with correction.

Reply:
I have a local fix which works for me, but I'd like to understand the history before pushing it. This PR seems to contain obvious bugs (and typos in the comments) and yet it was merged without question... so maybe I'm missing something?